proposals/infra/INF-0004-validator-hashing.md

```diff
@@ -123,42 +124,140 @@ should be that all shaders are validated and hashed using what is today the
 internal validator.
 
 For pre-release shader models DXC should run the validator, but it should not
-apply the hash. Instead it should apply the `PREVIEW` sentinel value from the
-table above. This allows us to differentiate between shaders validated by a
-"final" validator.
+apply the hash. Instead it should apply the `PREVIEW_BYPASS` sentinel value 
+from the table above. This allows us to differentiate between shaders validated 
+by a "final" validator.
+
+### IDxcValidator API Update
+
+A new `IDxcValidator` API flag is needed to avoid container hashing and
+unnecessary overhead when used by runtime validation.
+
+Proposed change to flags in `dxcapi.h`:
+
+```cpp
```

Comment:
If you specify this as `diff` instead of `cpp` the rendering will actually render as an inline unified diff.

Reply:
Or just strip it down as @tex3d suggested below (I guess I should read the other comments before commenting).